frontend/lib/src/features/user_profile/profile_info.dart

```diff
@@ -52,136 +71,158 @@ class ProfileInfo extends StatelessWidget {
       }
     }
 
-    return GetX<UserProfileController>(
-      builder: (profile) {
-        if (profile.isLoading) return const CircularProgressIndicator();
+    return FutureBuilder(
+      future: controller.getUser(userId: profileUserId ?? ''),
+      builder: (context, snapshot) {
+        if (snapshot.connectionState != ConnectionState.done) {
+          return const CircularProgressIndicator();
+        }
 
         return GestureDetector(
           onTap: () => FocusManager.instance.primaryFocus?.unfocus(),
           child: Column(
             children: [
-              const Breadcrumb(
+              Breadcrumb(
                 text: 'Profile Info',
-                withBackIcon: false,
+                withBackIcon: withBackIcon ?? false,
+                onTap: backIconOnTap,
               ),
-              Padding(
-                padding: const EdgeInsets.fromLTRB(25, 21, 25, 150),
-                child: FormBuilder(
-                  key: formKey,
-                  onChanged: () {
-                    formKey.currentState?.save();
-                    formKey.currentState?.fields['first_name']
-                        ?.validate(focusOnInvalid: false);
-                    formKey.currentState?.fields['last_name']
-                        ?.validate(focusOnInvalid: false);
-                    formKey.currentState?.fields['address']
-                        ?.validate(focusOnInvalid: false);
-
-                    controller.setFirstName =
-                        formKey.currentState?.fields['first_name']?.value ?? '';
-                    controller.setMiddleName =
-                        formKey.currentState?.fields['middle_name']?.value ??
+              Obx(
+                () {
+                  if (controller.isLoading) {
```

Comment:
remove "is"

rename to a more descriptive name
relate it with the submission of data